python/pyarrow/tests/test_ipc.py

```diff
@@ -240,6 +240,19 @@ def test_empty_stream():
         pa.ipc.open_stream(buf)
 
 
+def test_read_year_month_nano_interval(tmpdir):
```

Comment:
Can you add a reference to the JIRA here? Otherwise it's difficult to understand why this test is written this way.

Reply:
done.